apps/web/screens/Grants/SearchResults/SearchResults.tsx

```diff
@@ -98,6 +106,13 @@ const GrantsSearchResultsPage: CustomScreen<GrantsHomeProps> = ({
     })
   }, [])
 
+  const totalPages = useMemo(() => {
+    if (!totalHits) {
+      return
+    }
+    return totalHits > 8 ? Math.ceil(totalHits / PAGE_SIZE) : 1
```

Comment:
Maybe use your PAGE_SIZE constant here instead of the hardcoded 8?

Reply:
🙈